.speakeasy/overlay.mcp.yaml

```
@@ -26,18 +27,21 @@ actions:
       x-speakeasy-mcp:
         disabled: false
         name: list-incidents
+        scopes: "read"
 
   - target: $.paths..[?@.operationId == "create_incident"]
     update:
       x-speakeasy-mcp:
         disabled: false
         name: create-incident
+        scopes: "read", "write"
```

I _think_ yaml arrays have to either have brackets around them if they're inline:
```suggestion
        scopes: ["read", "write"]
```

Or they need to span multiple lines with ticks infront:
```suggestion
        scopes:
          - "read"
          - "write"
```